packages/eddsa-ticket-pcd/README.md

```diff
@@ -1,3 +1,192 @@
+        <img alt="Docs" src="https://img.shields.io/badge/docs-typedoc-purple.svg?style=flat-square">
+    </a>
+</p>
+
+| This package defines a PCD designed to verify the authenticity of an event ticket signed using an EdDSA key. |
+| ------------------------------------------------------------------------------------------------------------ |
+
+## 🛠 Install
+
+Install the `@pcd/eddsa-ticket-pcd` package with npm:
+
+```bash
+npm i @pcd/eddsa-ticket-pcd
+```
+
+or yarn:
+
+```bash
+yarn add @pcd/eddsa-ticket-pcd
+```
+
+## 📜 Usage
+
+### Prove
+
+```javascript
+import { newEdDSAPrivateKey } from "@pcd/eddsa-pcd"
+import { ITicketData, prove } from "@pcd/eddsa-ticket-pcd"
+import { ArgumentTypeName } from "@pcd/pcd-types"
+import { v4 as uuid } from "uuid";
+
+// Generate a new EdDSA private key.
+const privateKey = newEdDSAPrivateKey()
+
+// Prepare the event ticket to sign.
+// The following are just dummy values used for testing.
+const ticketData: ITicketData = {
+    attendeeName: "test name",
+    attendeeEmail: "user@test.com",
+    eventName: "event",
+    ticketName: "ticket",
+    checkerEmail: "checker@test.com",
+    ticketId: uuid(),
+    eventId: uuid(),
+    productId: uuid(),
+    timestampConsumed: Date.now(),
+    timestampSigned: Date.now(),
+    attendeeSemaphoreId: "12345",
+    isConsumed: false,
+    isRevoked: false,
+    ticketCategory: TicketCategory.Devconnect
+}
+
+// Create a PCD with the required attributes and their types.
+// It uses the EdDSA PCD to sign the message, which is the seralized ticket in this PCD.
+const pcd = await prove({
+    // The id is optional and if you don't pass it a random value will be automatically created.
+    id: {
+        argumentType: ArgumentTypeName.String
+    },
+    ticket: {
+        value: ticketData,
+        argumentType: ArgumentTypeName.Object
+    },
+    privateKey: {
+        argumentType: ArgumentTypeName.String,
+        value: privateKey
+    }
+})
+
+console.log(pcd)
+/*
+EdDSATicketPCD {
+  type: 'eddsa-ticket-pcd',
+  id: 'd79667d9-ed37-4e38-8e90-487628484fce',
+  claim: {
+    ticket: {
+      attendeeName: 'test name',
+      attendeeEmail: 'user@test.com',
+      eventName: 'event',
+      ticketName: 'ticket',
+      checkerEmail: 'checker@test.com',
+      ticketId: '69f7d074-cd96-4ffa-b406-f25ef6abccfd',
+      eventId: '22ee4549-fd4d-4814-85e7-d0e7077032f5',
+      productId: '9c9af161-6209-431a-953c-589604d5c5c4',
+      timestampConsumed: 1696776631601,
+      timestampSigned: 1696776631601,
+      attendeeSemaphoreId: '12345',
+      isConsumed: false,
+      isRevoked: false,
+      ticketCategory: 1
+    }
+  },
+  proof: {
+    eddsaPCD: EdDSAPCD {
+      type: 'eddsa-pcd',
+      id: '2aef80f0-bcdc-4075-b384-90948567cb80',
+      claim: [Object],
+      proof: [Object]
+    }
+  }
+}
+*/
+```
+
+### Verify
+
+```javascript
+import { verify } from "@pcd/eddsa-ticket-pcd"
+
+const isValid = await verify(pcd)
+
+console.log(isValid) // true
```

Comment:
worth communicating that just because a ticket pcd 'verifies', does not necessarily mean that it's the ticket that is actually needed to perform some sort of action. i recommend adding some comments and code describing that you need to check the event id and the product id, and the cases where you need to check those. another important thing to verify is that the public key of the entity that signed this ticket is actually the authority on that event - i.e. not just anybody can sign a ticket and have that be valid, but a specific public key was used.